spec/descriptions/putMaintenanceConfig.md

```diff
@@ -0,0 +1,44 @@
+This endpoint creates or updates a maintenance configuration given its ID.
+
+### Path Parameters:
+
+- **id:** The ID of the maintenance config to create or update.
+
+### Types of Maintenance Configurations
+- One-Time Maintenance Window
+- Recurrent Maintenance Window
+
+### **Recurrent Maintenance Window - RRULE Limitations**
+
+_NOTE: Not all `RRULE` options from the [iCalendar Spec](https://datatracker.ietf.org/doc/html/rfc5545) are supported._
```

Comment:
I would rephrase this as mention something like this:
Support for scheduling recurrent maintenance configurations is possible using the RRULE standard from the [iCalendar Spec](https://datatracker.ietf.org/doc/html/rfc5545). 
The following RRULE tokens are supported: "FREQ", "UNTIL", "COUNT", "INTERVAL", "BYDAY", "BYMONTHDAY", "BYMONTH". 

Then, just remove the `Supported` title, but keep the content